Mlem/App/Views/Shared/Labels/FullyQualifiedLabelView.swift

```diff
@@ -69,22 +69,38 @@ struct FullyQualifiedLabelView: View {
                 )
                 .frame(width: labelStyle.avatarSize, height: labelStyle.avatarSize)
             }
-            
+            let flairs = flairs
+            if !flairs.isEmpty {
+                HStack(spacing: 2) {
+                    ForEach(flairs, id: \.self) { flair in
+                        Image(systemName: flair.icon)
+                            .foregroundStyle(flair.color)
+                    }
+                }
```

Comment:
This should use the same rendering system as post tags (i.e., concatenated `Text` blocks)--it should be pretty easy to refactor this to use `postTag`, since that already just expects an icon and a color, and pass the resulting `Text` in to `FullyQualifiedNameView` to be prepended

Reply:
Sure, makes sense 👍